mvc/Tests/Security/SSOListenerTest.php

```diff
@@ -83,7 +83,7 @@ public function testGetPreAuthenticatedData()
 
         $userId = 123;
         $passwordHash = md5('password');
-        $legacyUser = new eZUser(array('contentobject_id' => $userId));
+        $legacyUser = @new eZUser(array('contentobject_id' => $userId));
```

Comment:
did not fix this as the aim is to only silence deprecation errors regarding constructor since that is the only one we most likely can't do anything about w/o introducing BC.

So left this one here but with a comment, which I guess is ok given this is test code only.
see 90d6969